### PR TITLE
PUPIL-284 -  quiz intro page

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

connect intro page to lesson data and display lesson info cards from oak components.

https://deploy-preview-2229--oak-web-application.netlify.thenational.academy/pupils/programmes/science-primary-ks1/units/growing-plants/lessons/what-plants-need-to-grow-and-stay-healthy

{owa_deployment_url}/pupils/programmes/english-secondary-ks4-aqa/units/macbeth-and-masculinity-the-struggle-for-power/lessons/duncan-as-a-father-figure
## How to test

1. Go to {owa_deployment_url}/pupils/programmes/science-primary-ks1/units/growing-plants/lessons/what-plants-need-to-grow-and-stay-healthy

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

### Acceptance criteria

- [x]  Available lesson info is displayed in fixed order
- [x]  logic to display default message when no info is available
- [x]  test file implemented
- [x]  remove from sonar cloud exclusions

### CODE REVIEW

File: src/components/PupilViews/PupilIntro/PupilIntro.view.tsx [hunk 1]
```diff
@@ -1,28 +1,189 @@
...
+    </OakLessonBottomNav>
+  );
+
+  const removedGuidanceDuplicates = Array.from(
+    new Set(
+      contentGuidance?.map(
+        (guidance: ContentGuidance) => guidance.contentGuidanceLabel,
+      ),
+    ),
+  );
```
Comment (carlmw) [inline src/components/PupilViews/PupilIntro/PupilIntro.view.tsx:76]: Annoying that this is a thing 😭

**Review by carlmw — APPROVED** [review]
Code LGTM, no notes.

File: src/components/PupilViews/PupilIntro/PupilIntro.view.tsx (hunk 1 above)
Comment (benprotheroe) [reply src/components/PupilViews/PupilIntro/PupilIntro.view.tsx:76]: as in the data should already be in a useable format?

**Review by Monika-Koziol — APPROVED** [review]

**Pull request merged**